File: sangi_practice/sangi_practice_01/model_answers/java/Main.java
import java.util.Scanner;

public class Main {
    public static void main(String[] args) {
        Scanner sc = new Scanner(System.in);
        int p = sc.nextInt(), n = sc.nextInt(), m = sc.nextInt();
        int total = p * n;
        int change = m - total;
        System.out.println(total);
        System.out.println(change);
    }
}

File: sangi_practice/sangi_practice_01/model_answers/c/main.c
#include <stdio.h>

int main(void) {
    int p, n, m, total, change;
    scanf("%d %d %d", &p, &n, &m);
    total = p * n;
    change = m - total;
    printf("%d\n%d\n", total, change);
    return 0;
}
